backend/src/api/routes/answers.py

```diff
@@ -92,14 +92,20 @@ async def save_survey_answer(
                 status_code=400, detail="User already answered this survey"
             )
 
+        message = (
+            "".join(question.answer for question in survey_answer.questions)
```

Comment:
This fails for multi and rank. Their answer field is a list, not a string.

Reply:
Done, we now concatenate the elements of the lists